### ISSUE TITLE
VoiceOver not working well with MdCards

### ISSUE THREAD

**Issue by evkw** [angular/material2#1128]
#### Bug, feature request, or proposal:

Bug
#### What is the expected behavior?

VoiceOver finds card headers, images and content and reads it
#### What is the current behavior?

VoiceOver cant find headers, images or content, only seems to read buttons
#### What are the steps to reproduce?

http://plnkr.co/edit/pkUNGMXPcf8RXKapXNXQ?p=preview
#### What is the use-case or motivation for changing an existing behavior?

Meet the a11y specification mentioned in the MD of this repo.
#### Which versions of Angular, Material, OS, browsers are affected?
- Angular2
- Material Design 2
- OSX El Capitan 10.11.2
- Chrome Version 52.0.2743.116 (64-bit)

**Comment by jelbourn** [comment]
This is likely a problem with the lack of real documentation presently. Right now, the different parts of the card are presentational only; it's still up to the application developer to use the appropriate `<h2>, `<p>`, etc. However, we should consider also allowing these the card-title, etc. to be applied as attributes in addition to container elements. 

Needs confirmation that it works as expected when using the appropriate elements.

**Comment by evkw** [comment]
Thanks for the reply.

I added a bunch of <p>s and <section>s to the card and it still didn't seem happy.

restaurant.component.html in https://github.com/evkw/Udacity.RestaurantReviewer if you need an example.

**Comment by jelbourn** [comment]
Closing this since I think this is now obsolete.
